### ISSUE TITLE
Simple localization of static pages

### ISSUE THREAD

**Issue by BinaryCraX** [webstudio-is/webstudio#3548]
As far as I know there are currently two options to localize a static page:
- Create a page in one language. After it is finished: duplicate the page and change all the texts to another language.
   This has the disadvantage that if a page is changed (e.g. layout/design), the change must be made for all languages.
- Create a dynamic page with a "/:lang"-variable in the path. This could than be used to fetch the localizations from an external source.
   This has the disadvantage, that you have to use a second tool just for localizations (of a otherwise static) sites. And that the initial setup per page is a little complicated.

Am I missing some option?
Is there a preferred way to accomplish this?

(rough) alternative suggestion:
- Allow languages to be added to a page in the page settings (under the current "Language"-field). This will automatically add a "/:lang"-variable to the path.
- Allow the languages to be checked as "shown"/published. To allow the user to work on a localization for a page while not publishing it yet.
- The resulting system-variable "system.params.lang" could be used to e.g. show the currently selected language in the top navbar.
- Add a system-variable "available-page-langs" (collection), which could be used to e.g. create a language selection in the top navbar, to switch the language on the currently viewed page.
- Currently viewed language in the designer could be selected through the existing UI at the top of the designer:
   ![grafik](https://github.com/webstudio-is/webstudio/assets/5560801/898907ef-bcef-4b84-ba35-4ea68c17a685)
- In the Binding-Popup add a shortcut for "Create Localized Text", which will create a new localization-key (changeable by the user) and allows you to edit the value (text) for each available language.
- Bonus: add a table view for all available localization per page/globally for easier bulk-editing:
  ![grafik](https://github.com/webstudio-is/webstudio/assets/5560801/ddf33393-1eed-4297-bd37-c04978b0b3bd)

**Comment by kof** [comment]
one other option that may be not a good fit for you depending on the content is using a content variable with all strings in both language, its basically like CMS but it would be static and local

**Comment by kof** [comment]
A better option will be available with nocode components. Where you can create a component out of all the pieces of your page and reuse it on multiple pages for different languages.

**Comment by thepra** [comment]
I agree, I was evaluating the software but missing localization that doesn't waste peoples time is crucial for me, so its a pass for me for now.

**Comment by kof** [comment]
created this issue to follow for the problem with accessing system variables from a slot, generally we expect people to use a cms for localization, static is possible but would involve lots of duplication, instances, so not really recommended

https://github.com/webstudio-is/webstudio/issues/4166
